.gitignore

```diff
@@ -1,32 +1,45 @@
-*~
-.DS_Store
+# Clojure / leiningen files
+pom.xml
+pom.xml.asc
+*.jar
+*.class
+/lib/
+/classes/
+/target/
+/checkouts/
 .lein-deps-sum
 .lein-repl-history
+.lein-plugins/
+.lein-failures
 .nrepl-port
-/pom.xml
-/pom.xml.asc
-/clojush*.jar
-classes/
-target/
+.cpcache/
+.lein-project-checksum
+
+# Editor and backup files
+*~
+.idea
+.gorilla-port
+.ipynb_checkpoints/
+.calva/
+.clj-kondo/
+.lsp
+
+# OS Files
+.DS_Store
+
+# Results and data directories
 results/
+/data/
+logs/
+
+# Unknown / Other
 .classpath
 .project
 .settings/ccw.repl.cmdhistory.prefs
-/data/
-logs/
-lib/
 /bin
-.lein-failures
 doc/
 cov/*
 out
-.idea
 clojush.iml
```

Comment:
This is configuration about a "module" of an Intellij project. Probably good that we are ignoring it given that we aren't in a team that is enforcing a single IDE.

Reply:
Yeah, this was moved above, not deleted.

Reply:
Sorry I should have specifically called out the line I was referring to because Github doesn't do a great job of it. 

I was referencing the `clojush.iml` line in the `Unknown` section.